argoverse/data_loading/stereo_dataloader.py

```diff
@@ -0,0 +1,107 @@
+# <Copyright 2021, Argo AI, LLC. Released under the MIT license.>
+
+import glob
+import sys
+from pathlib import Path
+from typing import Any, List, Mapping, Optional, Union
+
+import cv2
+import numpy as np
+
+from argoverse.utils.json_utils import read_json_file
+
+DISPARITY_NORMALIZATION: float = 2.0 ** 8  # 256.0
+
+
+class ArgoverseStereoDataLoader:
+    """
+    Stereo data loader for retrieving log data, given a path to the dataset.
+    """
+
+    def __init__(self, data_dir: str, split_name: str) -> None:
+        """
+        Args:
+            data_dir: Path to the Argoverse stereo data
+            split_name: Data split, one of train, val, or test
+        """
+        self.data_dir = data_dir
+        self.split_name = split_name
+
+    def get_log_calibration_data(self, log_id: str) -> Mapping[str, Any]:
+        """
+        Args:
+            log_id: Unique ID of vehicle log
+
+        Returns:
+            log_calib_data: calibration dictionary
+        """
+        calib_fpath = (
+            f"{self.data_dir}/rectified_stereo_images_v1.1/{self.split_name}/{log_id}/"
+            f"vehicle_calibration_stereo_info.json"
+        )
+
+        log_calib_data = read_json_file(calib_fpath)
+        assert isinstance(log_calib_data, dict)
+        return log_calib_data
+
+    def get_ordered_log_stereo_image_fpaths(self, log_id: str, camera_name: str) -> List[str]:
+        """
+        Args:
+            log_id: Unique ID of vehicle log
+            camera: camera based on camera_stats.RECTIFIED_STEREO_CAMERA_LIST
+
+        Returns:
+            stereo_img_fpaths: List of paths to chronologically ordered rectified stereo images in this log
+        """
+        stereo_img_fpaths = sorted(
+            glob.glob(f"{self.data_dir}/rectified_stereo_images_v1.1/{self.split_name}/{log_id}/{camera_name}/*.jpg")
+        )
+
+        return stereo_img_fpaths
+
+    def get_ordered_log_disparity_map_fpaths(self, log_id: str, disparity_name: str) -> List[str]:
+        """
+        Args:
+            log_id: Unique ID of vehicle log
+            disparity_name: disparity map name, one of:
+
+                ["stereo_front_left_rect_disparity",
+                 "stereo_front_left_rect_objects_disparity"]
+
+        Returns:
+            disparity_map_fpaths: List of paths to chronologically ordered disparity maps in this log
+        """
+        disparity_map_fpaths = sorted(
+            glob.glob(f"{self.data_dir}/disparity_maps_v1.1/{self.split_name}/{log_id}/{disparity_name}/*.png")
+        )
+
+        return disparity_map_fpaths
+
+    def get_rectified_stereo_image(self, stereo_img_path: str) -> np.ndarray:
+        """
+        Get the rectified stereo image.
```

Comment:
Should be `""" Get the rectified stereo image.` (on the same line as triple quote)